yaml/timber-ingressroute.yaml

```diff
@@ -44,6 +44,20 @@ spec:
         - name: timber-api-svc
           port: 3000
           weight: 10
+    - kind: Rule
+      #match requests to this domain
+      match: PathPrefix(`/ivr/`)
```

Comment:
In UAT, this is accessed via https://api.gov.yk.ca/timberreporting-UAT.

In production, this is accessed at https://api.gov.yk.ca/timberreporting-prod

@rsylvestre where do you think documentation about this setup go?